src/main/java/by/itech/lab/supplier/repository/UserRepository.java

```diff
@@ -16,7 +16,7 @@
     Optional<User> findById(Long id);
 
     @Modifying
-    @Query("update User set activeForDeletion = false where id = :id")
+    @Query("update User set deleted = false where id = :id")
```

Comment:
from the logic side: if `deleted == true` - its not deleted, but if `false` - its deleted. is it ok?

Reply:
Yes, we all should fix it in our classes. we'll talk about it tomorrow

Reply:
as per our discussion everyone who will change something near to places where its should be renamed - should rename it